internal/cli/server/flags.go

```diff
@@ -329,13 +329,13 @@ func (c *Command) Flags() *flagset.Flagset {
 		Group:   "JsonRPC",
 	})
 	f.SliceStringFlag(&flagset.SliceStringFlag{
-		Name:  "jsonrpc.corsdomain",
+		Name:  "http.corsdomain",
```

Comment:
Slightly sceptical about this change as the `corsdomain` and `vhosts` settings are applied to HTTP, WebSocket and GraphQL. If we change the prefix to `http` it might be misguiding and also with this prefix, the flag still lies in the `JsonRPC` struct in the config. 

There are 2 possibilities. Either we keep it as it was earlier (with prefix `jsonrpc`) and use it for HTTP, WS, and GraphQL. Or we add this as a flag in the `APIConfig` struct in config and then we'll have different flags for all 3 of them with their own prefix.

Reply:
Agree with Manav. I think we should go with the second option (adding this flag in `APIConfig` along with WS and GraphQL), since it aligns with geth's cli.